app/views/active_admin/shared/_theme_setup.erb

```diff
@@ -0,0 +1,8 @@
+<% # On page load or when changing themes, best to add inline in `head` to avoid FOUC %>
+<%= javascript_tag nonce: true do %>
+  if (localStorage.theme === 'dark' || (!('theme' in localStorage) && window.matchMedia('(prefers-color-scheme: dark)').matches)) {
```

Comment:
Should `theme` be prefixed by `activeAdmin` or configurable?

I understand that in some circumstances it is better to use the same application theme, but changing the theme in the admin area should not affect the application and vice versa 🤔

/cc @javierjulio